src/tequila/hamiltonian/custom_jw_transform.py

```diff
@@ -0,0 +1,237 @@
+import typing
+import openfermion
+from tequila.hamiltonian.qubit_hamiltonian import QubitHamiltonian
+
+
+def _custom_transform(fermion: str, qubits: list) -> QubitHamiltonian:
+    """
+    This method maps fermion to qubits.
+
+    Precondition:
+    The last index of qubits should be greater than
+    the site value of fermion.
+    For instance fermion: "3", qubits: [1, 2, 3, 4] is allowed, but
+    fermion: "3", qubits: [1, 2, 3] is not allowed
+
+    Post condition:
+    QubitHamiltonian of the qubits representation is returned
+
+    === Parameter ===
+    fermion: string representation of fermion operator to be mapped
+    qubits: list of qubits on which Paulis are applied
+    e.g. cjw.custom_transform("3^", [0, 1, 3, 5])
+
+    === Return ===
+    QubitHamiltonian
+    """
+    if not _is_input_compatible(fermion, qubits):
+        raise Exception("Incorrect Input Format")
+
+    qubit_hamiltonian = QubitHamiltonian()
+    operator = fermion[0]
+    site = qubits[int(operator[0])]
+    pauli_x = 'X' + str(site)
```

Comment:
can be removed after changes further below are done

Reply:
I removed parts where QubitHamiltonian can be applied